sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_events_v1/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+friendly_name: Feature Usage Events
+description: |-
+  Metrics from events pings for mobile feature usage dashboards (Fenix)
+owners:
+- rzhao@mozilla.com
+labels:
+  incremental: true
```

Comment:
Looking at the query, it's not 100% clear to me if this is intended to be an incremental query or of this is meant to be an overwrite table kind of query. Could you please confirm?